lale/lib/_common_schemas.py

```diff
@@ -176,6 +176,24 @@ def check_scoring_best_score_constraint(scoring=None, best_score=0) -> None:
     "default": 5,
 }
 
+schema_cv_1_1: JSON_TYPE = {
+    "description": "Determines the cross-validation splitting strategy used in cross_val_predict to train final_estimator.",
+    "anyOf": [
+        schema_simple_cv,
+        {
+            "enum": ["prefit"],
+            "description": '"prefit" to assume the estimators are prefit. In this case, the estimators will not be refitted.',
+            "forOptimizer": False,
+        },
+        {
+            "laleType": "CrossvalGenerator",
```

Comment:
Would it make sense to use `"laleType": "CrossvalGenerator"` not just for `schema_cv_1_1` but also for `schema_cv`?

Reply:
Yes, that's a good idea.